src/replica/testConfiguration.cc

```diff
@@ -95,8 +95,11 @@ BOOST_AUTO_TEST_CASE(ConfigurationTestStaticParameters) {
 
 BOOST_AUTO_TEST_CASE(ConfigurationInitTestJSON) {
     LOGS_INFO("Testing JSON initialization");
-
+#if 0
     BOOST_REQUIRE_NO_THROW(config = Configuration::load(ConfigTestData::data()));
+#else
+    config = Configuration::load(ConfigTestData::data());
+#endif
```

Comment:
Is is useful to hold onto the BOOST_RQUIRE_NO_THROW line?

Reply:
That was a leftover from some debugging effort. I will eliminate the second branch of the condition. Thank you for noticing it!